v4/export/sql.go

```diff
@@ -481,13 +481,7 @@ func resetDBWithSessionParams(db *sql.DB, dsn string, params map[string]interfac
 	}
 
 	for k, v := range support {
-		var s string
-		if str, ok := v.(string); ok {
-			s = wrapStringWith(str, "'")
-		} else {
-			s = fmt.Sprintf("%v", v)
-		}
-		dsn += fmt.Sprintf("&%s=%s", k, url.QueryEscape(s))
+		dsn += fmt.Sprintf("&%s=%s", k, url.QueryEscape(fmt.Sprintf("%v", v)))
```

Comment:
the `if` which was introduced in #111, it this now irrelevant with `--snapshot`?

Reply:
This commit has been reverted. I add two comments here.